crypto/edhoc-crypto-hacspec/src/lib.rs

```diff
@@ -77,6 +78,7 @@ pub fn aes_ccm_decrypt_tag_8(
 
     (err, p3)
 }
+
```

Comment:
remove newline :)

Reply:
This was actually intentional, since there was no blank line between the functions (and having such a blank line between functions seems to be an idiom in the codebase).

By the way, what do you think of using [rustfmt](https://doc.rust-lang.org/book/appendix-04-useful-development-tools.html) ?

Reply:
Sounds good! All for `rustfmt`, you can just issue `cargo fmt` command for it to do its job

Reply:
great